### PR TITLE
Add scripts to preprocess europarl corpus and generate alignments

### PR DESCRIPTION
**Pull request by sgoycoechea**

### CODE REVIEW

**Comment by bryant1410** [comment]
@sgoycoechea before I review, I think you changed the LF line-endings for CRLF (Windows ones), so it makes it hard to view the diff (everything looks like changed, but by looking closely, it isn't). Can you make sure all files follow LF line-endings?

**Comment by sgoycoechea** [comment]
Done

File: README.md [hunk 1]
```diff
@@ -105,20 +86,11 @@ Note you need a computer with a CUDA-capable GPU to train the model.
...
-./scripts/generate-alignments/generate_files.py
-spm_encode --model=en.model --output_format=piece < test.en > test.32k.en  
-spm_encode --model=es.model --output_format=piece < test.es > test.32k.es
-./scripts/generate-alignments/process_answer_indexes.py
-mkdir corpus-es
-mv test.32k.en test.32k.es answers.en corpus-es/
-./thualign/bin/generate.sh -s spanish -o output.txt
-spm_decode --model=es.model --input_format=piece < output.txt > output-plain.txt
-./scripts/generate-alignments/output_brackets_to_indexes.py
+./scripts/generate-alignments/generate_alignments.sh
```
Comment (bryant1410) [inline README.md:92]: Can you add the next part ("Generate the final merged CSV file") into this script as well?

File: README.md [hunk 2]
```diff
@@ -35,29 +35,10 @@ Alternatively, follow these steps to train it yourself.
 ### Prepare the Corpus
 
 1. Download the [Europarl Spanish-English parallel corpus](https://www.statmt.org/europarl/v7/es-en.tgz).
-2. Remove the sentences that don't form a pair (the sentences that correspond with an empty line).
-3. Remove sentences of length 1.
```
Comment (bryant1410) [inline README.md:39]: From above, not this line (I can't reference the proper line...), can you please change `conda env update` for `conda env create`?

**Review by bryant1410 — COMMENTED** [review]
I leave some comments for now, while I review the rest.

File: README.md [hunk 3]
```diff
@@ -35,29 +35,10 @@ Alternatively, follow these steps to train it yourself.
...
-  spm_encode --model=es.model --output_format=piece < validation.es > validation.32k.es
-  spm_encode --model=es.model --output_format=piece < test.es > test.32k.es
-  python thualign/scripts/shuffle_corpus.py --corpus corpus.32k.es corpus.32k.en
-  sed -i -e 's/<s>/<eos>/' -e 's/<s\/>/<pad>/' corpus.32k.en
-  sed -i -e 's/<s>/<eos>/' -e 's/<s\/>/<pad>/' validation.32k.en
-  sed -i -e 's/<s>/<eos>/' -e 's/<s\/>/<pad>/' test.32k.en
-  sed -i -e 's/<s>/<eos>/' -e 's/<s\/>/<pad>/' corpus.32k.es
-  sed -i -e 's/<s>/<eos>/' -e 's/<s\/>/<pad>/' validation.32k.es
-  sed -i -e 's/<s>/<eos>/' -e 's/<s\/>/<pad>/' test.32k.es
+./scripts/train-mask-align/preprocess_europarl.sh
```
Comment (bryant1410) [inline README.md:40]: What if you incorporate the corpus downloading into the script? Like:

```bash
wget -qO- https://www.statmt.org/europarl/v7/es-en.tgz | tar -xz -
```

**Review by bryant1410 — COMMENTED** [review]
I left another comment. Continue later.

File: README.md (hunk 1 above)
Comment (sgoycoechea) [reply README.md:92]: Done

File: README.md (hunk 3 above)
Comment (sgoycoechea) [reply README.md:40]: I implemented that but using a different command for tar, since using the line exactly like you sent it I got an error.

**Review by bryant1410 — APPROVED** [review]

**Pull request merged**